content/constants.js

```diff
@@ -1,12 +1,20 @@
+const fs = require("fs");
 const path = require("path");
 
 require("dotenv").config({
   path: path.join(__dirname, "..", process.env.ENV_FILE || ".env"),
 });
 
-const CONTENT_ROOT = process.env.CONTENT_ROOT
-  ? path.join(__dirname, "..", process.env.CONTENT_ROOT)
-  : path.join(__dirname, "..", "content", "files");
+const CONTENT_ROOT = (() => {
+  const root =
+    process.env.CONTENT_ROOT || path.join(__dirname, "..", "content", "files");
+  // If the CONTENT_ROOT wasn't an absolute (existing) directory, it's
+  // assumed to be relative to the project root.
+  if (!fs.existsSync(root)) {
+    return path.join(__dirname, "..", root);
+  }
+  return root;
+})();
```

Comment:
It works but it's ugly. 
The only reason it is like this is because of workspaces and stuff and because we want to be able to express things like `export CONTENT_ROOT=content/files` which needs to be "fixed".